internal/provider/fwprovider/provider.go

```diff
@@ -142,6 +144,18 @@ func (p *fwprovider) Schema(ctx context.Context, req provider.SchemaRequest, res
 					},
 				},
 			},
+			"auth_login":            AuthLoginGenericSchema(),
```

Comment:
Is there a reason we can't use the auth login registry as before? That solved some issues where not all expected AuthLogin were properly set.

Reply:
Good question, we might need to use that. I will look into it. Thanks!

Reply:
I took a closer look at this again.

These lines define the schema for the Framework Provider (which must match 1-1 with the sdkv2 Provider). The auth login registry is still being used by the Framework Provider. The entrypoint for the auth login registry is in [setClient](https://github.com/hashicorp/terraform-provider-vault/blob/852ea9a1c2e48079c6fe7ad101654b90795fb576/internal/provider/meta.go#L177):

https://github.com/hashicorp/terraform-provider-vault/blob/852ea9a1c2e48079c6fe7ad101654b90795fb576/internal/provider/meta.go#L255

As such, there should be no behavior change for the Framework Provider since we are sharing the auth login code with the sdkv2 provider. Please let me know if you need more clarification!